subworkflows/local/utils_nfcore_seqinspector_pipeline/main.nf

```diff
@@ -84,8 +84,8 @@ workflow PIPELINE_INITIALISATION {
         .fromSamplesheet("input") // Validates samplesheet against $projectDir/assets/schema_input.json. Path to validation schema is defined by $projectDir/nextflow_schema.json
         .map {
             meta, fastq_1, fastq_2 ->
-                def id_string = "${meta.sample}_${meta.group ?: "ungrouped"}_${meta.lane}"
-                def updated_meta = meta + [ id: id_string ]
+                def tags = meta.tags ? meta.tags.tokenize(",") : []
```

Comment:
```suggestion
                def tags = meta.tags ? meta.tags.tokenize(",").trim().toLowerCase() : []
```
Not sure if we should enforce lower case in tags, but we are using the tags as part of the file name and some file systems, e.g. that of MacOS, are case-insensitive. So either we enforce the conversion to make sure that we do not accidentally overwrite files or we need to have a check function that searches for identical tags like "`batch3,Batch3`" and raises an error.

Reply:
> or we need to have a check function that searches for identical tags like batch3 & Batch3 and raises an error.

This seems like the nicest approach to me, since we are likely to want to retain tags like `P12345` as uppercase at NGI at least.

Preventing the possibility of overwriting things, while still maintaining the user's preference of tag names seems like a good compromise.